src/Raven.Client/Documents/Operations/Replication/PullReplicationDefinition.cs

```diff
@@ -80,4 +82,12 @@ public ExternalReplication ToExternalReplication(ReplicationInitialRequest reque
             };
         }
     }
+
+    [Flags]
+    public enum PreventDeletionsMode
+    {
+        Disabled,
```

Comment:
Default value should be `None`

```suggestion
        None,
```

Reply:
Fixed